File: Palindrome/Python/ivanchoff/palindrome.py
def isPalindrome(cad):
    size_cad = len(cad)
    for i in range(int(size_cad/2)):
        if cad[i] != cad[size_cad - i - 1]:
            return False
    return True

if __name__ == '__main__':
    words = ['hello','world','ana','alula']
    for i in words:
        print('is word ',"'",i,"' Palindrome? ",isPalindrome(i))

File: Palindrome/javascript/ivanchoff/palindrome.js
const isPalindrome = function(cad) {
  size_cad = cad.length;
  for(let i=0; i<size_cad/2; i++){
    if(cad[i] != cad[size_cad-i-1])
      return 0;
  }
  return 1;
};

module.exports = isPalindrome;
